### PR TITLE
Remove deprecated SVG helpers for old IEs

### PR DESCRIPTION
**Pull request by ChristophWurst**
Removed long deprecated APIs and fixed usage (noops).

@MorrisJobke please double-check for any usage in 3rd party apps. Thanks!

### CODE REVIEW

File: core/js/js.js [hunk 1]
```diff
@@ -1981,33 +1981,6 @@ OC.Util = {
 		}
 		return moment(timestamp).fromNow();
 	},
-	/**
-	 * Returns whether the browser supports SVG
-	 * @deprecated SVG is always supported (since 9.0)
-	 * @return {boolean} true if the browser supports SVG, false otherwise
-	 */
-	hasSVGSupport: function(){
```
Comment (MorrisJobke) [inline core/js/js.js]: https://github.com/nextcloud/passman/blob/c0de19650a6f5398fbfd443240f955ffb3b9e80d/tests/unit/js/mocks/OC.js#L1508

Comment (MorrisJobke) [reply core/js/js.js]: Same file for the other two methods.

**Review by MorrisJobke — APPROVED** [review]

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Could you add this to https://github.com/nextcloud/server/issues/11045?

File: core/js/js.js (hunk 1 above)
Comment (ChristophWurst) [reply core/js/js.js]: Submitted at https://github.com/nextcloud/passman/issues/506.

**Pull request merged**